scripts/prepare_doc_publication.py

```diff
@@ -123,6 +123,11 @@ def parse_arguments():
     'GetStarted': 'Getting Started',
 }
 
+# A dictionary containing the supporting JavaScript files for each doc.
+JS_FILES_DICT = {
+  'op_coverage.md': ['js/add_classes.js']
+}
```

Comment:
This will work for now, but we could also specify frontmatter in the markdown files themselves, though that's a larger change to this file.

Reply:
That would be nice. The way I put this in there was a bit hacky.